tests/caasp/transactional_update.pm

```diff
@@ -81,7 +81,10 @@ sub run {
 
     # Check that zypper does not return 0 if update was aborted
     record_info 'Broken pkg', 'Install broken package poo#18644 - snapshot #3';
-    if (is_caasp('DVD')) {
+    if (is_caasp('=4.0')) {
+        record_info 'Test skipped - broken image needs breaking again';
```

Comment:
is this a bug (ergo bsc?)? or expected behaviour?

Reply:
thanks for merge :) - this is bug in tests, not product bug.
We have package that should fail during zypper install. Now it installs fine cause package specfile needs updating.

I will fix that when I am back from vacation, I just didn't want test to fail until then.